configs/kubernetes-nightly-configmap.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
 data:
   config: |
     target-org: kubernetes-nightly
-    github-issue: 56876
-    dry-run: false
\ No newline at end of file
+    # github-issue: 56876
```

Comment:
This was to test if the issue reporter was working, right? Can we delete it now?

Reply:
Yes, the nightly config will go away.